python/example_code/sns/sns-python-example-publish-to-topic.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+import boto3
+
+# Create an SNS client
+sns = boto3.client('sns')
+
+# Publish a simple message to the specified SNS topic
+response = sns.publish(
+    TopicArn='my-topic-arn',    

```

Comment:
you should put a demo arn here

```suggestion
    TopicArn='arn:aws:sns:region:0123456789:my-topic-arn',    
```